src/main/kotlin/org/jetbrains/gradle/ext/IdeaModelUtils.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.gradle.ext
```

Comment:
i was under the impression that these extension functions needed to be in the `org.gradle.kotlin.dsl` package. why did you place them in this package?

Reply:
This is just the unique package that gets exposed from within this plugin, I'm opened to any suggestion though

Reply:
@eskatos can you clarify why this belongs in `org.jetbrains.gradle.ext` and not `org.gradle.kotlin.dsl`?

Reply:
Because this is this plugin's package. Putting plugin code in `org.gradle.kotlin.dsl` in order for them to be available without import is an abusive workaround for the lack of https://github.com/gradle/gradle/issues/7557.

Reply:
Ah, I didn't know that. Thanks @eskatos !